lab_1_classify_by_unigrams/main.py

```diff
@@ -19,6 +28,15 @@ def calculate_frequencies(tokens: list[str] | None) -> dict[str, float] | None:
     :param tokens: a list of tokens
     :return: a dictionary with frequencies
     """
+    frequencies = {}
```

Comment:
Don't forget to add checks for incorrect types here and in other functions.

Reply:
Done.